linked_list/linked_list.cpp

```diff
@@ -0,0 +1,325 @@
+/*
+* C++ Program to Implement Linked List
+*/
+#include <iostream>
+#include <cstdio>
+#include <cstdlib>
+
+using namespace std;
+/*
+* Node Declaration
+*/
+struct node
+{
+	int info;
+	struct node *next;
+}*start;
+
```

Comment:
@mohitkyadav The guides says that structs should be used for data only whereas classes for other cases. By using structs,I am following the guidlines.